backend/app/models/user.rb

```diff
@@ -19,11 +19,11 @@ def self.find_by_credentials(username, password)
     user = User.find_by(username: username)
     return nil unless user
 
-    user.password?(password) ? user : nil
+    user.is_password?(password) ? user : nil
   end
 
-  def password?(password)
-    BCrypt::Password.new(password_digest).password?(password)
+  def is_password?(password)
+    BCrypt::Password.new(password_digest).is_password?(password)
```

Comment:
Wait which is right? `password?` or `is_password?`? Was it broken before or now?